### DISCUSSION TITLE
Make the disableExtensions option disable only h1 instead of all headings

### DISCUSSION THREAD

**Discussion by destroyer22719** [outline/rich-markdown-editor#553] [Ideas]
Hi there! After looking at the code here
https://github.com/outline/rich-markdown-editor/blob/18f519f2337b8f8f1010afa398223f57867fc04d/src/index.tsx
On line 103 the only option is to disable `heading` which will disable ALL headings. Whereas I only want to disable `h1` but not the other `h2`, `h3`

**Comment by tommoor** [comment]
Hi, try `headingsOffset` prop. 

https://github.com/outline/rich-markdown-editor#headingsoffset
